README.md

```diff
@@ -15,9 +15,9 @@ Plugin                          | Slug                      | Experimental | Lin
 [Modern Image Formats][2]       | `webp-uploads`            | No           | [Source][10], [Issues][18], [PRs][26]
 [Performant Translations][3]    | `performant-translations` | No           | [Source][11], [Issues][19], [PRs][27]
 [Speculative Loading][4]        | `speculation-rules`       | No           | [Source][12], [Issues][20], [PRs][28]
-[Embed Optimizer][5]            | `embed-optimizer`         | Yes          | [Source][13], [Issues][21], [PRs][29]
+[Embed Optimizer][5]            | `embed-optimizer`         | No           | [Source][13], [Issues][21], [PRs][29]
+[Image Prioritizer][7]          | `image-prioritizer`       | No           | [Source][15], [Issues][23], [PRs][31]
```

Comment:
Nit-pick, but we may want to keep these alphabetically sorted (I.e. integrate in the list items above), with experimental plugins at the end.

Reply:
Yes, good point. Fixed in b8d1019bd285ab29b94e09d83b2f6161e52ec317